app/models/conversation.rb

```diff
@@ -1,13 +1,17 @@
 # frozen_string_literal: true
 
 class Conversation < ApplicationRecord
-  has_many :messages, dependent: :destroy
+  has_many :messages, -> { order :created_at }, dependent: :destroy, inverse_of: :conversation
```

Comment:
No need to add this order, by default records are ordered by id which is the same as ordering by creation

Reply:
Also I think`inverse_of` is not necessary as it seems to be necessary only if you are modifying something from the options of the association (primary key, table name, etc...)

Reply:
`inverse_of` was something rubocop complained about. So I had to add it. I'm not sure why tho. Maybe rubocop got messed up because of the scope in here. I'll update.